Create Issue Templates (#5278)
ECP-WarpX/WarpX: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest a new feature or enhancement.
labels: [enhancement]
---

> 

Reviewer: ```suggestion

_Please remove any sensitive information (e.g., passwords, API keys) from your submission. Thank you for taking the time to report this issue. We will respond as soon as possible._

```

> ---
> name: Feature request
> about: Request a new feature
> title: ''
> labels: [enhancement]
> assignees: ''
> ---

Reviewer: ```suggestion
---
name: Feature request
about: Suggest a new feature or enhancement.
labels: [enhancement]
---
```

**What is the context that is prompting you to request a new feature?**
A clear and concise description of the context.

**What feature would you like to add to WarpX?**
A clear and concise description of the feature.

**What alternatives have you considered?**
A clear and concise description of any alternative solutions or features you have considered.

**Additional information**

> If applicable, please add other info about the feature request.

Reviewer 2: Maybe here or in the "what feature" section above we should ask for a publication reference if the new feature is a model found in other codes?

Reviewer: ```suggestion
**Additional information**
If applicable, please add other info about the feature request (e.g., links to existing codes, publications, etc.).
```
